<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMaterialsDialog</class>
 <widget class="QDialog" name="AddMaterialsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="cmbBox_Types">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lst_AudioFormats">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>151</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lst_BonusFeatures">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>151</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_Frame">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_AddAudioFormat">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_AddBonusFeature">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_DelBonusFeature">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>350</y>
     <width>21</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_DelAudioFormat">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>21</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_AudioFormats">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>151</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_BonusFeatures">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>151</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Types">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Select a Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Frame">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Frame</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Audo Formats</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Bonus Features</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Apply">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>400</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Cancel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <zorder>cmbBox_Types</zorder>
  <zorder>lst_AudioFormats</zorder>
  <zorder>lst_BonusFeatures</zorder>
  <zorder>txt_Frame</zorder>
  <zorder>Btn_AddBonusFeature</zorder>
  <zorder>btn_DelBonusFeature</zorder>
  <zorder>btn_DelAudioFormat</zorder>
  <zorder>txt_AudioFormats</zorder>
  <zorder>txt_BonusFeatures</zorder>
  <zorder>btn_AddAudioFormat</zorder>
  <zorder>lbl_Types</zorder>
  <zorder>lbl_Frame</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>btn_Apply</zorder>
  <zorder>btn_Cancel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
